test/test_pmm.py

```diff
@@ -814,6 +829,18 @@ def test_config_spread_on_the_fly_multiple_orders(self):
         self.assertAlmostEqual(Decimal("96"), last_bid_order.price, 2)
         self.assertAlmostEqual(Decimal("104"), last_ask_order.price, 2)
 
+    def test_order_override(self):
+        strategy = self.one_level_strategy
```

Comment:
strategy here is `order_override_strategy` that you created, so you haven't tested anything of your changes actually.

Reply:
Yes, I've changed it to order_override_strategy.